repos/system_upgrade/el7toel8/actors/repositoriesmapping/actor.py

```diff
@@ -0,0 +1,36 @@
+import csv
+
+from leapp.actors import Actor
+from leapp.models import RepositoriesMap, RepositoryMap
+from leapp.tags import IPUWorkflowTag, ChecksPhaseTag
+
+
+class RepositoriesMapping(Actor):
+    """
+    Produces message containing repository mapping based on provided file.
+    """
+
+    name = 'repository_mapping'
+    consumes = ()
+    produces = (RepositoriesMap,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+
+    
+    def process(self):
+        repos_maps = RepositoriesMap()
+
+        with open(self.get_file_path('repomap.csv')) as f:
+            data = csv.reader(f)
+            for row in data:
+                if len(row) != 6:
+                    continue
+                
+                from_uid, to_uid, from_minor_version, to_minor_version, arches, repo_type = row
+
+                repos_maps.repositories.append(RepositoryMap(from_uid=from_uid,
```

Comment:
If I'm not mistaken (and I tried to google it), it is not repository **UID** but **ID**. I changed it in the source sheet. UID from what I know stands for user ID. Please change it to _id_ wherever you use _uid_.

Reply:
@bocekm I guess in the meaning here it is unique ID. I will try to find that again, but I believe the UID it is used here. In same source it is called even "label" and not ID.

Reply:
It's possible that I proposed UID as I found in the past references like: unique repository ID (where UID is more accurate from my point of view). However, in other source is used just "repo ID" (e.g. `yum repolist all`). So not sure what to use, but in case we want to use something else then UID, it should be changed everywhere - see `targetrepositories.py`. So the same thing will not use various names. So you can
- rename it to something else - then use same naming everywhere
- keep it as it is now and create new PR to change this everywhere at once later
- you can rename it here and create new PR for already existing code

..whatever.

Reply:
I am not marking this one as resolved, but I followed Michal suggestion and renamed UID to ID for this PR. I did it mostly since "repo ID" is used inside yum. But, as Petr mentioned, there are many places where UID is used. Let me know if I should keep ID for this PR and rename everything else in a follow up PR or something else.

Reply:
Yeah, keep ID in this PR and create a new PR to change it on the other places. Thanks.